packages/lib/gpc/test/gpc.spec.ts

```diff
@@ -694,7 +694,8 @@ describe("gpc library (Compiled test artifacts) should work", async function ()
           entries: ["pod1.$signerPublicKey", "pod2.$signerPublicKey"],
           isMemberOf: "admissiblePubKeyPairs"
         }
-      }
+      },
+      uniquePODs: true
```

Comment:
I think it's okay not to test both uniquePODs=true and uniquePODs=false at this level, so long as it was tested at the gpcircuits level.